daemon/anonymous-statistics.sh.in

```diff
@@ -22,8 +22,8 @@ if [ -f "@configdir_POST@/.opt-out-from-anonymous-statistics" ] || [ ! "${DO_NOT
   exit 0
 fi
 
-# Shorten version for easier reporting
-NETDATA_VERSION=$(echo "${NETDATA_VERSION}" | sed 's/-.*//g' | tr -d 'v')
+NETDATA_IS_IN_CI="false"
+[ -n "${CI}" ] && NETDATA_IS_IN_CI="true"
```

Comment:
@Ferroin 

Will the node's environment variables be propagated to the docker container?

Reply:
I guess not and that means we have problems with `NETDATA_IS_IN_CI`

Reply:
Hmm, hi Ilya, should we remove it then.... Or is there another way we can get the variable... ?

Reply:
I don't see how `$CI` can be set when executing `anonymous-statistics.sh`. Not a specialist, let's wait for @Ferroin reply.

Reply:
@Ferroin , please, can you give your opinion here?

Reply:
Hi! We could just merge this, and check the results :-) If all we get is falses, we can re-check or remove it. We'll make sure it doesn't get into a dashboard, @andrewm4894 are you ok with this?

Reply:
Sorry about the delayed response, this somehow slipped through the cracks.

In practice, this will only work if Netdata is running natively in the CI environment. It won’t work in containers regardless of the container management stack (part of the point of a container is to be decoupled from the host environment), and it may not work in other circumstances as well (for example, it may not work if netdata is started by something that scrubs the environment, like for example `sudo`).

Reply:
> In practice, this will only work if Netdata is running natively in the CI environment.

So we need to remove the variable from the script/analytics, am I correct? It won't the way it is implemented.

Reply:
If that is the case, let's take it out so we can merge this!

Reply:
OK, removed!